### ISSUE TITLE
LinkML documentation show allow for some flexibility in which annotations appear in a permissible value table

### ISSUE THREAD

**Issue by turbomam** [linkml/linkml#1605]
NMDC has some permissible values with `meanings` that show up in the tabular view of the enum.

The `meanings` annotations really make more sense as `see_also`s. But we would still want to see them in the table.

@aclum and I would either like some CLI flag for re-configuring the tables, or some clear documentation on how to do it in the Jinja templates.

**Comment by cmungall** [comment]
I suggest the default is simply to show every populated metaslot, with a priority order determined by the rank in the metamodel. This would satisfy most cases with minimal need for bespoke configuration

**Comment by sujaypatil96** [comment]
I agree with the general approach of showing all metamodel slots that have been populated in the schema. Modifying the base jinja templates to accommodate that will take a little more dedicated thought/effort.

In the meantime, since this issue is sort of NMDC-specific we can even modify the custom jinja templates that we have in the [nmdc-schema](https://github.com/microbiomedata/nmdc-schema) repo to allow the annotations to be rendered in the permissible values table as well.

**Comment by sujaypatil96** [comment]
Consider for example an enum in the nmdc-schema: https://microbiomedata.github.io/nmdc-schema/BiosampleCategoryEnum/

Question: rendering all metaslots of permissible values and showing them in the table here: https://microbiomedata.github.io/nmdc-schema/BiosampleCategoryEnum/#permissible-values will make it a wide-scrollable table – are we okay with compromising on the real estate/changing layout/aesthetics to accomplish this?

**Comment by turbomam** [comment]
action item for @turbomam : write up recent developments in class/slot flattening. Do we want to have some general tool that does the same thing for enumerations and PVs?

sample output:

https://github.com/microbiomedata/external-metadata-awareness/blob/main/unorganized/linkml-meta-schema.tsv
